builtin/providers/ns1/permissions.go

```diff
@@ -0,0 +1,184 @@
+package ns1
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/account"
+)
+
+func addPermsSchema(s map[string]*schema.Schema) map[string]*schema.Schema {
+	s["dns_view_zones"] = &schema.Schema{
+		Type:     schema.TypeBool,
+		Optional: true,
+	}
+	s["dns_manage_zones"] = &schema.Schema{
+		Type:     schema.TypeBool,
+		Optional: true,
+	}
+	s["dns_zones_allow_by_default"] = &schema.Schema{
```

Comment:
Should be fixed on master now if you want to rebase.

You should probably drop your vendor commit, and re-add those packages correctly using govendor after rebasing, as that will fail a check now too.